frontend/src/fetchers/event.js

```diff
@@ -18,6 +19,30 @@ export const updateEvent = async (id, data) => {
   return res.json();
 };
 
+export const completeEvent = async (id, data) => updateEvent(id, {
+  ...data,
+  data: {
+    ...data.data,
+    status: TRAINING_REPORT_STATUSES.COMPLETE,
+  },
+});
+
+export const suspendEvent = async (id, data) => updateEvent(id, {
+  ...data,
+  data: {
+    ...data.data,
+    status: TRAINING_REPORT_STATUSES.SUSPENDED,
+  },
+});
+
+export const resumeEvent = async (id, data, status) => updateEvent(id, {
+  ...data,
+  data: {
+    ...data.data,
+    status,
```

Comment:
Is this right? The test for `resumeEvent` doesn't make use of this third `status` argument, so I think this ends up setting `status` to `undefined`, which might not be what we're looking for. Should it be `TRAINING_REPORT_STATUSES.COMPLETE`?

```javascript
  it('completes the event on the server with the given id', async () => {
    const event = await resumeEvent('1', { ownerId: 1, regionId: 1, data: {} });

    expect(fetchMock.called()).toBe(true);
    expect(fetchMock.lastUrl()).toBe('/api/events/id/1');
    expect(event).toEqual({ id: 1, name: 'test event' });
  });
```